saleor/plugins/openid_connect/plugin.py

```diff
@@ -163,9 +175,14 @@ def external_authentication(
             return previous_value
         storefront_redirect_url = data.get("redirectUrl")
         validate_storefront_redirect_url(storefront_redirect_url)
+        kwargs = {
+            "redirect_uri": build_absolute_uri(f"/plugins/{self.PLUGIN_ID}/callback"),
```

Comment:
In the good old Django views we used to use `reverse` function to get URLs by name, can we use it here as well?